src/Symfony/Component/Translation/Loader/PoFileLoader.php

```diff
@@ -12,15 +12,15 @@
 namespace Symfony\Component\Translation\Loader;
 
 /**
- * @copyright Copyright (c) 2010, Union of RAD http://union-of-rad.org (http://lithify.me/)
+ * @copyright Copyright (c) 2010, Union of RAD http://union-of-rad.org (https://lithiumseo.com/)
```

Comment:
I would replace the 2 URLs by the Github one instead: https://github.com/UnionOfRAD/lithium

Reply:
Copy that!